db/views/materialized_patient_summaries_v03.sql

```diff
@@ -0,0 +1,416 @@
+with latest_bp_passport as (
+    select distinct on (patient_id) *
+    from patient_business_identifiers
+    where identifier_type = 'simple_bp_passport' and deleted_at is null
+    order by patient_id, device_created_at desc
+), latest_phone_number as (
+    select distinct on (patient_id) *
+    from patient_phone_numbers
+    where deleted_at is null
+    order by patient_id, device_created_at desc
+), latest_medical_history as (
+    SELECT DISTINCT ON (patient_id) * FROM medical_histories WHERE deleted_at IS NULL
+),ranked_prescription_drugs as (
+    select
+        bp.id as bp_id,
+        prescription_drugs.*,
+        rank() over (partition by bp.id order by prescription_drugs.is_protocol_drug desc, prescription_drugs.name, prescription_drugs.device_created_at desc) rank
```

Comment:
Ordering here by device_created_at so that more recently prescribed drugs are more likely to be pushed into the numbered columns, instead of being grouped in the `Other Prescription Drugs` column.